src/android/CodePush.java

```diff
@@ -616,6 +631,37 @@ public void onStart() {
         }
     }
 
+    private Boolean hasIonicWebViewEngine() {
+        try {
+            Class.forName("com.ionicframework.cordova.webview.IonicWebViewEngine");
+            return true;
+        } catch (ClassNotFoundException e) {}
+        return false;
+    }
+
+    private void setServerBasePath(String serverPath) {
+        try {
+          Class ionicWebViewEngineClass = Class.forName("com.ionicframework.cordova.webview.IonicWebViewEngine");
+          Object ionicWebViewEngine = ionicWebViewEngineClass.cast(this.mainWebView.getEngine());
+          Method setServerBasePath = ionicWebViewEngineClass.getMethod("setServerBasePath", String.class);
+
+          final String ionicWebViewEngineServerPath = serverPath;
```

Comment:
Can we just add `final` to method argument for preventing declaring additional variable?